kokoro/build.sh

```diff
@@ -32,7 +32,7 @@ function install_requirements() {
 
 function run_unit_tests() {
     echo Running unit tests.
-    python -m pytest dataflux_core/tests --junit-xml="${KOKORO_ARTIFACTS_DIR}/unit_tests/sponge_log.xml"
```

Comment:
verbose flag would be nice to include, but not strictly necessary. Its useful in the case of a freeze or failure so that we can still see individualized sub-test output.

Reply:
Will keep it